sim/src/mechanics/driving.rs

```diff
@@ -233,7 +235,7 @@ impl DrivingSimState {
             let mut car = self.cars.remove(&id).unwrap();
             // Responsibility of update_car_with_distances to manage scheduling stuff!
             if self.update_car_with_distances(
-                &mut car, &dists, idx, now, map, parking, trips, scheduler, transit, walking,
+                &mut car, &dists, idx, now, map, parking, trips, scheduler, transit, walking, cap,
```

Comment:
I'm extremely unhappy with all of the parameter plumbing everywhere. We can't just pass `&mut Sim`, because we're in the middle of borrowing a piece of it (`DrivingSimState` in this case). I wonder if there's a nice way to express "all of `Sim` except the current piece"

Reply:
https://old.reddit.com/r/rust/comments/idjcua/passing_all_other_fields_of_a_struct_to_one/

Maybe someone'll have a good idea